presto-main/src/main/java/com/facebook/presto/server/QueryResource.java

```diff
@@ -68,16 +69,16 @@ public QueryResource(QueryManager queryManager, AccessControl accessControl, Ses
     }
 
     @GET
-    public List<BasicQueryInfo> getAllQueryInfo()
+    public Map<String, BasicQueryInfo> getAllQueryInfo()
```

Comment:
Do we need this change? It seems like the client can just index the infos on the client side, if they want a map